policy-gh-action-dependencies/action.yml

```diff
@@ -1,22 +1,23 @@
-name: 'kubewarden-policy-gh-action-dependencies'
-description: 'Install all the binaries needed inside of GH action'
+name: "kubewarden-policy-gh-action-dependencies"
+description: "Install all the binaries needed inside of GH action"
 branding:
-  icon: 'package'
-  color: 'blue'
+  icon: "package"
+  color: "blue"
 runs:
   using: "composite"
   steps:
-    -
-      name: Install cosign
+    - name: Install cosign
       uses: sigstore/cosign-installer@v3
-    -
-      name: Install kwctl
+    - name: Install kwctl
       uses: kubewarden/github-actions/kwctl-installer@v3.1.12
-    -
-      name: Install bats
+    - name: Install bats
       uses: mig4/setup-bats@v1
       with:
         bats-version: 1.8.2
-    -
-      name: Install SBOM generator tool
+    - name: Install SBOM generator tool
       uses: kubewarden/github-actions/sbom-generator-installer@v3.1.12
+    - name: Install wasm-opt
+      shell: bash
+      run: |
+        apt-get update
+        apt-get install -y binaryen
```

Comment:
this installs an old version of wasm-opt ( I think 105), that had some issue with a memory leak. For the kyverno dsl policy I ended up installing a newer version from the github releases.

Reply:
Good point! I've changed  that